package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```

Comment:
This is mostly to support webpack 2.4.1, which we're using at twitter due to https://github.com/webpack/webpack/issues/4850 in 2.5.1 and above